README.md

```diff
@@ -120,7 +120,7 @@ spack:
   finalImage: "ubuntu:22.04"
   processorTarget: "x86_64_v3"
 
-coreURL: "http://x.y.z:9837/upload"
+coreURL: "http://x.y.z:9837/softpack"
```

Comment:
So the upload endpoint is /softpack/upload?

Reply:
Yes; we have two different Core endpoints we now use 'upload' and 'graphql', so it makes sense for the coreURL to point at the base path for the two.